x-pack/plugins/xpack_main/server/routes/api/v1/settings.js

```diff
@@ -7,6 +7,7 @@
 import { wrap as wrapError } from 'boom';
 import { KIBANA_SETTINGS_TYPE } from '../../../../../monitoring/common/constants';
 import { getKibanaInfoForStats } from '../../../../../monitoring/server/kibana_monitoring/lib';
+import { getEmailValueStructure } from '../../../../../monitoring/server/kibana_monitoring/collectors/get_settings_collector';
```

Comment:
we should be able to not have this import when `getEmailValueStructure` is a method of settingsCollector